nfc/labAttendancy.py

```diff
@@ -18,6 +20,8 @@ def on_connect(tag):
     sc = nfc.tag.tt3.ServiceCode(service_code >> 6, service_code & 0x3f)
     bc0 = nfc.tag.tt3.BlockCode(0, service=0)
 
+    beep(1567,300)
```

Comment:
ここをスレッド化しないと300ms? たつまで学籍番号を読み取ってくれない

```suggestion
    beep_thread = threading.Thread(
        target = beep,
        args = (1567, 300)
    )
    beep_thread.start()
```